app/ViewLocations.tsx

```diff
@@ -0,0 +1,44 @@
+import { ReactElement } from "react";
+import { Wrapper } from "./ui/Wrapper";
+import { styled } from "styled-components";
+
+const locations = [
+  {
+    displayName: "TANG",
+    fullName: "TANG - The Asian Food Emporium",
+    suburb: "Melbourne",
+  },
+  {
+    displayName: "Minh Phat",
+    fullName: "Minh Phat Asian Grocer",
+    suburb: "Richmond",
+  },
+  {
+    displayName: "Aldi",
+    fullName: "ALDI",
+    suburb: "Richmond",
+  },
+];
+
+export function ViewLocations(): ReactElement {
+  const rowSeed = locations.map((location) => {
+    return (
+      <div style={{ display: "flex" }}>
+        <Cell style={{ color: "black" }}>{location.displayName}</Cell>
+        <Cell style={{ color: "#535353" }}>{location.suburb}</Cell>
+      </div>
+    );
+  });
+  return <Wrapper>{rowSeed}</Wrapper>;
+}
+
+const Cell = styled.div`
+  width: 48vw;
+  height: 49px;
+  font-size: 20px;
+  border-width: 0px;
+  border-bottom-width: 2px;
+  border-style: solid;
+  background-color: #00c2ff;
+  padding: 5px;
```

Comment:
this is some pretty weird styling but we can look at it together, feels like the width would be just 50%

Reply:
https://github.com/auberginearms/groceryDatabaseApp/pull/12/commits/da15f699dce5bd208883210c89b5a40008135b1a